ledger/participant-integration-api/src/main/resources/db/migration/oracle-appendonly/V1__Append_only_schema.sql

```diff
@@ -18,13 +18,13 @@ CREATE TABLE packages
     upload_id          NVARCHAR2(1000)            not null,
     -- A human readable description of the package source
     source_description NVARCHAR2(1000),
-    -- The size of the archive payload (i.e., the serialized DAML-LF package), in bytes
+    -- The size of the archive payload (i.e., the serialized Daml-LF package), in bytes
     package_size       NUMBER                     not null,
     -- The time when the package was added
     known_since        NUMBER                     not null,
     -- The ledger end at the time when the package was added
     ledger_offset      VARCHAR2(4000)             not null,
-    -- The DAML-LF archive, serialized using the protobuf message `daml_lf.Archive`.
+    -- The Daml-LF archive, serialized using the protobuf message `daml_lf.Archive`.
```

Comment:
Touching database migration files could cause issues as their content is hashed and check by the migration tool (that's why these files are excluded from copyright notice changes -- notice it still says 2021 up on top).

Reply:
We should probably not touch this file.

Reply:
I tried very hard to avoid those. I avoided so many of them!

Will restore that one. Thanks for noticing!